crates/dbs-boot/src/aarch64/layout.rs

```diff
@@ -80,3 +82,20 @@ pub const IRQ_MAX: u32 = dbs_arch::gic::IRQ_MAX;
 pub const MAPPED_IO_START: u64 = dbs_arch::gic::GIC_REG_END_ADDRESS; // 1 GB
 /// End address (inclusive) of the MMIO window.
 pub const MAPPED_IO_END: u64 = (2 << 30) - 1; // 1 GB
+
+lazy_static! {
+    /// Maximum guest physical address supported.
```

Comment:
no need for lazy static here, it's a special requirement of x86.

Reply:
done, I have changed it like this:
 pub static GUEST_PHYS_END: &u64 = &((1u64 << 40) - 1);